setup.py

```diff
@@ -67,7 +67,7 @@ def long_description():
     ],
     python_requires='>=3.5, <3.8',
     keywords='',
-    packages=find_packages(exclude=['test', 'test.*']),
+    packages=find_packages(exclude=['tests', 'tests.*']),
```

Comment:
@neiljp I tried this and this doesn't work! I can still see
```
creating 'dist/zulip_term-0.4.0rc0-py3-none-any.whl' and adding 'build/bdist.macosx-10.9-x86_64/wheel' to it
adding 'tests/__init__.py'
adding 'tests/conftest.py'
adding 'tests/cli/__init__.py'
adding 'tests/cli/test_run.py'
adding 'tests/core/__init__.py'
adding 'tests/core/test_core.py'
adding 'tests/helper/__init__.py'
adding 'tests/helper/test_helper.py'
adding 'tests/model/__init__.py'
adding 'tests/model/test_model.py'
adding 'tests/ui/__init__.py'
adding 'tests/ui/test_ui.py'
adding 'tests/ui/test_ui_tools.py'
adding 'tests/ui/test_utils.py'
adding 'zulipterminal/__init__.py'
adding 'zulipterminal/core.py'
```
while running `python3 setup.py sdist bdist_wheel`.
Perhaps we need to something they mentioned [here](https://stackoverflow.com/questions/8556996/setuptools-troubles-excluding-packages-including-data-files/11669299#11669299)
```quote
To also exclude the tests folder from source distributions, add the following to MANIFEST.in:

recursive-exclude tests *
```

Reply:
@amanagr I don't see this; I had only tested with `build`, but this seems to also be reflected in what I see with the results of your commands:
```bash
$unzip -v dist/zulip_term-0.4.0rc0-py3-none-any.whl 
Archive:  dist/zulip_term-0.4.0rc0-py3-none-any.whl
 Length   Method    Size  Cmpr    Date    Time   CRC-32   Name
--------  ------  ------- ---- ---------- ----- --------  ----
       0  Defl:N        2   0% 2018-06-19 04:40 00000000  zulipterminal/__init__.py
   12159  Defl:N     2640  78% 2019-07-18 00:27 13c5b98c  zulipterminal/core.py
   13988  Defl:N     3917  72% 2019-07-18 01:05 da169b35  zulipterminal/helper.py
   34048  Defl:N     8038  76% 2019-07-18 00:53 e3bcd7a8  zulipterminal/model.py
    8351  Defl:N     2356  72% 2019-07-21 03:27 03816285  zulipterminal/ui.py
      24  Defl:N       26  -8% 2019-07-21 03:34 3c874465  zulipterminal/version.py
       0  Defl:N        2   0% 2019-06-19 02:51 00000000  zulipterminal/cli/__init__.py
   10059  Defl:N     3140  69% 2019-07-17 19:17 01cde708  zulipterminal/cli/run.py
       0  Defl:N        2   0% 2019-06-19 02:51 00000000  zulipterminal/config/__init__.py
    6339  Defl:N     1556  76% 2019-07-18 00:27 2042fa77  zulipterminal/config/keys.py
    8240  Defl:N     1411  83% 2019-07-17 22:07 d5f5c60e  zulipterminal/config/themes.py
       0  Defl:N        2   0% 2019-02-19 22:29 00000000  zulipterminal/ui_tools/__init__.py
   33523  Defl:N     7140  79% 2019-07-18 00:53 bec4ae6d  zulipterminal/ui_tools/boxes.py
    8977  Defl:N     2297  74% 2019-07-18 00:53 eac3fad7  zulipterminal/ui_tools/buttons.py
    2455  Defl:N      872  65% 2019-07-18 00:27 7b5e09a4  zulipterminal/ui_tools/utils.py
   31115  Defl:N     6257  80% 2019-07-21 03:34 1d69433c  zulipterminal/ui_tools/views.py
   11357  Defl:N     3948  65% 2019-07-21 18:38 7b5d04bc  zulip_term-0.4.0rc0.dist-info/LICENSE
   18748  Defl:N     5923  68% 2019-07-21 18:38 9399820e  zulip_term-0.4.0rc0.dist-info/METADATA
      92  Defl:N       92   0% 2019-07-21 18:38 26c5d90e  zulip_term-0.4.0rc0.dist-info/WHEEL
      59  Defl:N       55   7% 2019-07-21 18:38 21202343  zulip_term-0.4.0rc0.dist-info/entry_points.txt
      14  Defl:N       16 -14% 2019-07-21 18:38 6b3acb0c  zulip_term-0.4.0rc0.dist-info/top_level.txt
       1  Defl:N        3 -200% 2019-07-21 04:17 32d70693  zulip_term-0.4.0rc0.dist-info/zip-safe
    1952  Defl:N      972  50% 2019-07-21 18:38 3d783dd1  zulip_term-0.4.0rc0.dist-info/RECORD
--------          -------  ---                            -------
  201501            50667  75%                            23 files
```
Also
```bash
$tar ztvf dist/zulip-term-0.4.0rc0.tar.gz |cut -d':' -f2|cut -d' ' -f2
zulip-term-0.4.0rc0/
zulip-term-0.4.0rc0/PKG-INFO
zulip-term-0.4.0rc0/README.md
zulip-term-0.4.0rc0/setup.cfg
zulip-term-0.4.0rc0/zulip_term.egg-info/
zulip-term-0.4.0rc0/zulip_term.egg-info/SOURCES.txt
zulip-term-0.4.0rc0/zulip_term.egg-info/PKG-INFO
zulip-term-0.4.0rc0/zulip_term.egg-info/dependency_links.txt
zulip-term-0.4.0rc0/zulip_term.egg-info/top_level.txt
zulip-term-0.4.0rc0/zulip_term.egg-info/zip-safe
zulip-term-0.4.0rc0/zulip_term.egg-info/entry_points.txt
zulip-term-0.4.0rc0/zulip_term.egg-info/requires.txt
zulip-term-0.4.0rc0/setup.py
zulip-term-0.4.0rc0/zulipterminal/
zulip-term-0.4.0rc0/zulipterminal/core.py
zulip-term-0.4.0rc0/zulipterminal/cli/
zulip-term-0.4.0rc0/zulipterminal/cli/run.py
zulip-term-0.4.0rc0/zulipterminal/cli/__init__.py
zulip-term-0.4.0rc0/zulipterminal/config/
zulip-term-0.4.0rc0/zulipterminal/config/keys.py
zulip-term-0.4.0rc0/zulipterminal/config/themes.py
zulip-term-0.4.0rc0/zulipterminal/config/__init__.py
zulip-term-0.4.0rc0/zulipterminal/helper.py
zulip-term-0.4.0rc0/zulipterminal/version.py
zulip-term-0.4.0rc0/zulipterminal/ui_tools/
zulip-term-0.4.0rc0/zulipterminal/ui_tools/buttons.py
zulip-term-0.4.0rc0/zulipterminal/ui_tools/views.py
zulip-term-0.4.0rc0/zulipterminal/ui_tools/boxes.py
zulip-term-0.4.0rc0/zulipterminal/ui_tools/utils.py
zulip-term-0.4.0rc0/zulipterminal/ui_tools/__init__.py
zulip-term-0.4.0rc0/zulipterminal/__init__.py
zulip-term-0.4.0rc0/zulipterminal/ui.py
zulip-term-0.4.0rc0/zulipterminal/model.py